mem0/memory/graph_memory.py

```diff
@@ -67,9 +71,13 @@ def add(self, data, filters):
 
         update_memory_prompt = get_update_memory_messages(search_output, extracted_entities)
 
+        _tools=[UPDATE_MEMORY_TOOL_GRAPH, ADD_MEMORY_TOOL_GRAPH, NOOP_TOOL]
+        if self.llm_provider == ["azure_openai_structured","openai_structured"]:
```

Comment:
this should be `if self.llm_provider in ["azure_openai_structured","openai_structured"]:`

Reply:
@spike-spiegel-21 can you please check this?

Reply:
done @prateekchhikara